packages/python-packages/protocol-stub-generator/protocol/protocol_models.py

```diff
@@ -434,31 +476,7 @@ def from_yaml(cls, yaml_data: Dict[str, Any], op_group_num, op_num, namespace):
                     namespace,
                 )
             )
-        for j in range(
-            0,
-            len(
-                yaml_data["operationGroups"][op_group_num]["operations"][op_num][
-                    "requests"
-                ]
-            ),
-        ):
-            for i in range(
-                0,
-                len(
-                    yaml_data["operationGroups"][op_group_num]["operations"][op_num][
-                        "requests"
-                    ][j].get("signatureParameters", [])
-                ),
-            ):
-                param.append(
-                    ProtocolParameterView.from_yaml(
-                        yaml_data["operationGroups"][op_group_num]["operations"][
-                            op_num
-                        ]["requests"][j],
-                        i,
-                        namespace,
-                    )
-                )
```

Comment:
just want to check that this delete block was intentional?

Reply:
Yes, that just was when I passed requests into parameters